webpack/assets/javascripts/react_app/common/MountingService.js

```diff
@@ -6,18 +6,6 @@ export { default as registerReducer } from '../redux/reducers/registerReducer';
 
 const mountedNodes = [];
```

Comment:
Should we keep managing this list now?
The whole point of this list was to be able to `unmount` all the `mountedNodes` before turbolinks mount them again.